Resources/Prototypes/_RMC14/Roles/Jobs/Survivor/colonist.yml

```diff
@@ -1,8 +1,8 @@
 ﻿- type: job
   parent: CMJobBase
-  id: CMSurvivor
-  name: cm-job-name-survivor
```

Comment:
Are they actually called colonists in 13? I can't find them on my local server so I can't check, but from the code and wiki they seem to be called survivors

Reply:
oh theres... many of them
```
#define JOB_COLONIST "Colonist"
#define JOB_PASSENGER "Passenger"
#define JOB_SURVIVOR "Survivor"
#define JOB_SYNTH_SURVIVOR "Synth Survivor"
#define JOB_CO_SURVIVOR "CO Survivor
```

Reply:
gonna role back to survivor then.